src/Components/CreateImageWizardV2/CreateImageWizard.tsx

```diff
@@ -148,6 +150,12 @@ const CreateImageWizard = ({ startStepIndex = 1 }: CreateImageWizardProps) => {
     selectBlueprintDescription(state)
   );
   const activationKey = useAppSelector((state) => selectActivationKey(state));
+  const partitions = useAppSelector((state) => selectPartitions(state));
+  const [isNextDisabled, setNextDisabled] = useState(false);
+
+  useEffect(() => {
+    setNextDisabled(false);
+  }, [partitions, dispatch]);
```

Comment:
I think that we can drop this lower into the child `<FileSystemStepFooter>` component.